packages/web-react/src/components/FileUploader/README.md

```diff
@@ -469,39 +469,98 @@ and a collection of props selected above for easier implementation. Other props
+| `validationText`      | \[`string` \| `string[]`]               | —        | ✕        | Validation status text                                                                                                        |
 
 On top of the API options, the components accept [additional attributes][readme-additional-attributes].
 If you need more control over the styling of a component, you can use [style props][readme-style-props]
 and [escape hatches][readme-escape-hatches].
 
 For detailed information see [FileUploader][file-uploader] component.
 
+### Understanding File Size in Bytes
+
+File size is measured in bytes, where larger units such as **kibibytes** (KiB) or **mebibytes** (MiB) are derived as follows:
+
+- 1 KiB (kibibyte) = 1,024 bytes
+- 1 MiB (mebibyte) = 1,024 KiB = 1,024 × 1,024 bytes = 1,048,576 bytes
+- 10 MiB (mebibytes) = 10 × 1,048,576 bytes = 10,485,760 bytes
+
+When setting the `maxFileSize` parameter, specify the maximum size of an uploaded file in bytes.
+
+For example:
+To limit files to 1 MiB, set `maxFileSize` to **1048576**.
+To limit files to 10 MiB, set `maxFileSize` to **10485760**.
+
+#### Kilo/Mega vs. Kibi/Mebi
+
+In computing, file sizes can be measured using two different standards.
+The **decimal** (base-10) and **binary** (base-2) systems.
+This can cause confusion when terms like `kilobyte` and `kibibyte` are used interchangeably.
+
+**Kilobyte** (kB): Based on the decimal system, 1 kilobyte equals 1,000 bytes.
+This is commonly used in contexts like hard drive capacities, where manufacturers use the decimal standard.
+
+**Kibibyte** (KiB): Based on the binary system, 1 kibibyte equals 1,024 bytes.
+This is often used in computer memory and file size calculations.
+
+The same distinction applies to larger units like **megabytes** (MB) and **mebibytes** (MiB):
+
+- 1 MB = 1,000,000 bytes (decimal)
+- 1 MiB = 1,048,576 bytes (binary)
+
+For precise calculations, especially when using parameters like `maxFileSize`,
+**it is recommended to use the binary standard** and specify values in bytes based on the binary system.
+
+For more details about binary prefixes and file size standards, check out [this Wikipedia article on binary prefixes][wiki-binary-prefixes].
+
+#### PHP & Next.js Configurations
+
+In some cases, you can set file sizes using shorthand notations. However, these values are still interpreted as binary sizes internally.
+
+For example, [in PHP](https://www.php.net/manual/en/ini.core.php#ini.upload-max-filesize) you can use the shorthand `1M` for the `upload_max_filesize`, which corresponds to **1,048,576 bytes (1 MiB)**:
+
+```php
+upload_max_filesize=1M
+```
+
+Similarly, [in Next.js](https://nextjs.org/docs/app/api-reference/config/next-config-js/serverActions#bodysizelimit), you can specify file size limits using a shorthand like `1mb`.
+This will also be interpreted as **1,048,576 bytes (1 MiB)**:
+
+```jsx
+module.exports = {
+  experimental: {
+    serverActions: {
+      bodySizeLimit: '1mb',
```

Comment:
Is the unit really all lowercase? I'd consider dropping a comment on it because the case is important for IT size units.

Reply:
This is copied right from the docs: https://nextjs.org/docs/app/api-reference/config/next-config-js/serverActions#bodysizelimit

🤷 

And that is why I asked @pavelklibani to make it more accurate and with examples. Because this is happening

🤷